caster-editor/src/components/NodeEditor.vue

```diff
@@ -56,8 +55,23 @@ const scriptCells = computed({
 }
 
 .editor-header-spacer {
-    width: inherit;
-    height: calc($menuHeight*2);
-    margin-bottom: 30px;
-  }
+  width: inherit;
+  height: calc($menuHeight*2);
+  margin-bottom: 30px;
+}
+
+// transition
+.slide-enter-active {
+  transition: all 0.3s cubic-bezier(0.215, 0.610, 0.355, 1.000); /* easeOutCubic */
+}
+
+.slide-leave-active {
+  transition: all 0.3s cubic-bezier(0.550, 0.055, 0.675, 0.190); /* easeInCubic */
+}
+
+.slide-enter-from,
+.slide-leave-to {
+  transform: translateX(900px);
+  opacity: 1;
+}
```

Comment:
why is the `slide` animation class attached to the component if the component doesn't know that it can slide? Shouldn't this be where "please slide in on transition" is declared, so in `GraphDetailView`?

Reply:
Not sure I understand what you mean :)